lib/common/client/middleware/response/snakecase.rb

```diff
@@ -24,9 +24,18 @@ def underscore_symbolize(hash)
             hash.deep_transform_keys { |k| k.underscore.to_sym }
           end
         end
+
+        class SnakecaseString < Snakecase
+          private
+
+          def underscore_symbolize(hash)
+            hash.deep_transform_keys(&:underscore)
```

Comment:
I tried using `with_indifferent_access` but there are tests that fail if you change it unilaterally, since they expect the keys to be symbols only and not interchangeable with strings.